MLS.Agent/Markdown/LocalCodeFenceAnnotationsParser.cs

```diff
@@ -113,7 +113,7 @@ private static void AddSourceFileOption(Command csharp)
                 var projectFiles = directoryAccessor.GetAllFilesRecursively()
                                                     .Where(file =>
                                                     {
-                                                        return directoryAccessor.GetFullyQualifiedPath(file.Directory).FullName == rootDirectory.FullName && file.Extension == ".csproj";
+                                                        return directoryAccessor.GetFullyQualifiedPath(file.Directory).FullName == rootDirectory.FullName && (file.Extension == ".csproj" || file.Extension == ".fsproj");
```

Comment:
There's a potential inconsistency here in that I can do this and it will be valid:

````markdown
```c# --project ./something.fsproj
```
````

Reply:
My plan (hope?) was to merge as-is to `feature/fsharp` (where this PR is targeted) and then do a merge in from `master` to get @colombod's changes that add a language parameter to the workspace object and clean up the various hacks I added around detecting the language.